ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__metal-4.16-latest-x86.yaml

```diff
@@ -1,15 +1,13 @@
+base_images:
+  cli:
+    name: "4.16"
+    namespace: ocp
+    tag: cli
```

Comment:
@paigerube14 Should this stanza be added to [openshift-qe-ocp-qe-perfscale-ci-main__metal-dev-latest-x86.yaml](https://github.com/openshift/release/pull/54209/files#diff-037a720d3172d519ddc777caa9092c55d10f8b4b32367e6c7d697aa8149111d7) as well?

Reply:
@paigerube14 I found there are some config files that do not have base_images stanza, like but not limited to 4.18 azure and gcp. I think we should add this to all files.
BTW: If they don't have it they will pick the latest cli version, correct?

Reply:
If I understand you correctly, the  `cli: latest` in a ref file matches the `latest` in the config file under `release` like below. 
```
releases:
  latest:
    candidate:
      product: ocp
      stream: nightly
      version: "4.17"
```

Reply:
And the base_images as in the config file like [aro-4.13-nightly-x86-upi.yaml](https://github.com/openshift/release/blob/a2c6f473cf5b0178415351b645257db1c53262c4/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly-x86-upi.yaml)
```
base_images:
  cli:
    name: "4.13"
    namespace: ocp
    tag: cli
```
 is for some other ref file that has `from: cli` like [ipi-install-rbac-ref.yaml](https://github.com/openshift/release/blob/a2c6f473cf5b0178415351b645257db1c53262c4/ci-operator/step-registry/ipi/install/rbac/ipi-install-rbac-ref.yaml#L3)